<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASChReDialog</class>
 <widget class="QDialog" name="ASChReDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>维修进度</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>报修人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>报修人登录名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOwnerUid">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>房号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtHouseId">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtKind">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtProblem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>派工</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtStaff">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtTele">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>当前状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtState">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>改变状态</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>待处理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>已派工</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>已解决</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>评价</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtEvaluate">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>星级</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtStar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>★★★★★</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtStaff</tabstop>
  <tabstop>edtTele</tabstop>
  <tabstop>boxState</tabstop>
  <tabstop>btnSubmit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnSubmit</sender>
   <signal>clicked()</signal>
   <receiver>ASChReDialog</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submit()</slot>
 </slots>
</ui>
